Package.swift

```diff
@@ -1,9 +1,9 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.9
```

Comment:
I had to bump this version to allow visionOS support. I'm not sure if this will have any unintended side effects on consumers though.